### PR TITLE
New Homepage Commissioners section

### PR DESCRIPTION
**Pull request by xtine**
Issue #639 

screen shot 2016-12-15 at 5 18 08 pm
screen shot 2016-12-15 at 5 19 06 pm

Relies on https://github.com/18F/fec-style/pull/586

### CODE REVIEW

File: fec/home/templates/home/home_page.html [hunk 1]
```diff
@@ -74,6 +74,75 @@
...
+        <p>Established in 1975, the FEC is composed of six Commissioners who are appointed by the President and confirmed by the Senate. By law, no more than three can represent the same political party.</p>
+        <a href="http://www.fec.gov/about.shtml">Learn more about the FEC’s history and mission »</a>
+      </div>
+
+      <div class="grid grid--3-wide grid--no-border">
+        <div class="grid__item">
+          <div class="icon-heading">
+            <img class="icon-heading__image" src="{% static "img/headshot--peterson.png" %}" alt="Headshot of Matthew S. Peterson">
+            <div class="icon-heading__content">
+              <div class="t-lead t-ruled-bottom"><a href="http://www.fec.gov/members/petersen/petersen.shtml">Matthew S. Peterson</a></div>
```
Comment (noahmanger) [inline fec/home/templates/home/home_page.html]: Why do you need the `t-ruled-bottom` class? Doesn't the `<a>` come with underlining?

Comment (xtine) [inline fec/home/templates/home/home_page.html]: Good catch, I had that on there before I added the links and forgot to take them out.

**Comment by xtine** [comment]
I prefer a simpler approach to just having the html markup for the commissioners as-is. I think there would be a case of storing it somewhere else if they were referenced on different pages or sections, but if it's just on the homepage then I don't see the need.

**Comment by ccostino** [comment]
Thanks, @xtine!  Merging in since there are no blockers here. :-)

**Pull request merged**